include/roboteam_ai/stp/new_plays/BallPlacementUs.h

```diff
@@ -1,11 +1,15 @@
+#ifndef RTT_BALLPLACEMENTUS_H
+#define RTT_BALLPLACEMENTUS_H
```

Comment:
How does this .h  file work without including Play.h?

Reply:
Its included in the cpp

Reply:
But everywhere else we include the parent class in the .h file

Reply:
Fixed it :)